### DISCUSSION TITLE
How to solve Stripe error.

### DISCUSSION THREAD

**Discussion by PaePongsiriTri** [Glench/ExtPay#75] [General]
Hi I need help I got this error when I try to pay by myself. I use bank in Thailand.
Stripe doesn't currently support application fees for platforms in US with connected accounts in TH.

I found this maybe it fix on platform or what should I do?
https://discourse.webflow.com/t/stripe-payment-failed/108943

**Comment by Glench** [comment]
Email me from the email address you used to sign up for ExtensionPay and I can fix this for you right away :) glen@extensionpay.com

**Reply by PaePongsiriTri** [reply]
Sent it! thank Mr.Glench.

**Comment by adamity** [comment]
Hye Glench, I've got the same issue with my Malaysia currency, I've emailed you my details, I hope you can help me with this, thanks Glench
